.standard.yml

```diff
@@ -1,3 +1,4 @@
 ignore:
   - 'config/environments/staging.rb'
-  - 'config/environments/production.rb'
\ No newline at end of file
+  - 'config/environments/production.rb'
+  - 'lib/tasks/deployment/20201005203405_populate_case_contact_contact_type.rake'
```

Comment:
we should fix this

Reply:
I thought about doing it, but it is a one off after_party task, that probably only got run once.

Reply:
@jimnanney kind of- it got run once on qa, once (successfully on staging, and will on friday be run (hopefully only once) on production. Plus it will be run on every new developer's laptop, once each- until we remove the need for it somehow, probably via smarter db seeding. I'm not too worried about it but it does need to keep working. There's an issue open to write a test for it also